### PR TITLE
VZ-5679 Ginkgo Test for Bom Validations Under Verify Install   

### PR DESCRIPTION
**Pull request by absBitm**
added ginkgo test for BOM validations that will run post vz installations verifications.

### CODE REVIEW

File: tests/e2e/verify-install/bom-validator/bom_validator_test.go [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+	if kubeconfig != "" {
+		fmt.Println("USING KUBECONFIG: ", kubeconfig)
+		return true
+	}
+	return false
+}
+
+// Get the BOM from the platform operator in the cluster and build the BOM structure from it
+func getBOM() {
+	var platformOperatorPodName string = ""
```
Comment (michael-cico) [inline tests/e2e/verify-install/bom-validator/bom_validator_test.go:130]: Are we getting rid of the bom-valdiator standalone tool?  If we are, I'm OK with this, but if we're keeping it the validation logic should be shared between the test impl and the standalone tool.

File: tests/e2e/verify-install/bom-validator/bom_validator_suite_test.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import (
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"testing"
+)
+
+func TestBomValidator(t *testing.T) {
+	gomega.RegisterFailHandler(ginkgo.Fail)
+	ginkgo.RunSpecs(t, "Bom Validator Suite")
```
Comment (kminder) [inline tests/e2e/verify-install/bom-validator/bom_validator_suite_test.go:14]: Changes all references to the acronym BOM to all caps.

File: tests/e2e/verify-install/bom-validator/bom_validator_test.go [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+	out, err := exec.Command("kubectl", "get", "pod", "-o", "name", "--no-headers=true", "-n", "verrazzano-install").Output()
```
[the remark this answers is not in the document]
Comment (absBitm) [reply tests/e2e/verify-install/bom-validator/bom_validator_test.go]: replaced all here [5ee5360](https://github.com/verrazzano/verrazzano/pull/3519/commits/5ee53600ebea8517b761d662ac0a95abb637e124)

File: tests/e2e/verify-install/bom-validator/bom_validator_suite_test.go [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020, 2022, Oracle and/or its affiliates.
```
[the remark this answers is not in the document]
Comment (absBitm) [reply tests/e2e/verify-install/bom-validator/bom_validator_suite_test.go]: done

**Review by kminder — DISMISSED** [review]
Approved.

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**